### PR TITLE
Added new table component

### LINKED ISSUE #308: Create a new table component
Create a new table component and add it to Design system. Please see design in Zeplin global styleguide https://app.zeplin.io/styleguide/5d95b0aaad77af16c796a476/components?seid=5d9af57dcbfaab1ebb7c8a87

Table variants

- [ ] Table / 2 col / 1 row
- [ ] Table / 2 col / 2 row
- [ ] Table / 2 col / 3 row
- [ ] Table / 2 col / 4 row
- [ ] Table / 3 col / 1 row
- [ ] Table / 3 col / 2 row
- [ ] Table / 3 col / 3 row
- [ ] Table / 3 col / 4 row

### PR DESCRIPTION
**Pull request by houbly**
Fixes #308

### CODE REVIEW

File: src/www/pages/components/table/index.njk [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+<h3>
+  When to use it?
+</h3>
+<ul>
+  <li>
+    Use the table component to let users compare information in rows and columns.
+  </li>
+  <h3>
+    When not to use it?
+  </h3>
```
Comment (gldgrnt) [inline src/www/pages/components/table/index.njk:24]: Duplicate title

**Review by gldgrnt — CHANGES_REQUESTED** [review]
All good except for one thing in the component description.

**Review by gldgrnt — APPROVED** [review]
Looking good!

**Review by mrmjprice — CHANGES_REQUESTED** [review]
Please can we vertical-align centre all table cells?

A design error. Apologies.

Kat and Jake will get this updated in Sketch/Zeplin.

**Comment by houbly** [comment]
> Please can we vertical-align centre all table cells?
> 
> A design error. Apologies.
> 
> Kat and Jake will get this updated in Sketch/Zeplin.

I was hoping someone would say this haha

**Comment by mrmjprice** [comment]
> > Please can we vertical-align centre all table cells?
> > A design error. Apologies.
> > Kat and Jake will get this updated in Sketch/Zeplin.
> 
> I was hoping someone would say this haha

If you spot something like this - raise it and throw it back to design. The system belongs to all of us. 😄

**Review by mrmjprice — DISMISSED** [review]

**Review by gldgrnt — DISMISSED** [review]
All good

File: src/wmnds/assets/sass/_vars.scss [hunk 2]
```diff
@@ -18,6 +18,7 @@ $palettes: (
   information: #84329b,
   disable: #676869,
   plannedDisruption: #ffdd00,
+  border: #979797,
```
Comment (catiarodriguescosta) [inline src/wmnds/assets/sass/_vars.scss:21]: Will this color be reused?
If so, you can let it here, if not, move it to the component own scss file.

Comment (houbly) [reply src/wmnds/assets/sass/_vars.scss:21]: There is an accordion header in the zeplin doc (listed with the tables) that uses it but I'm not sure when that will be needed. I'll just put it in the component scss for now as you suggested.

**Pull request merged**